fastlane_core/spec/configuration_file_spec.rb

```diff
@@ -64,12 +64,6 @@
         expect(config[:another_boolean]).to be(true)
       end
 
-      it "ignores the same item being set after the first time" do
```

Comment:
I was wondering about this, I'll take another stab at it, but I believe (and may be wrong) is that Appfile is not behaving in this same manner, and that was why I looked at removing this.

I'll see what I can do to get this back in there.